lib/fluent/plugin/output.rb

```diff
@@ -768,19 +767,13 @@ def extract_placeholders(str, chunk)
               end
             end
           end
+        end
 
-          if rvalue =~ CHUNK_KEY_PLACEHOLDER_PATTERN
-            log.warn "chunk key placeholder '#{$1}' not replaced. template:#{str}"
-          end
-
-          rvalue.sub(CHUNK_ID_PLACEHOLDER_PATTERN) {
```

Comment:
This patch has a problem with `@chunk_key_time`.
How about moving this code before `if rvalue =~ CHUNK_KEY_PLACEHOLDER_PATTERN` check?

Reply:
Done.